lib/Mir.ml

```diff
@@ -101,35 +84,48 @@ and 's statement =
    We'll need to scan the list of declarations for top-level ones and essentially remove them
    from the block. The AST has an `is_global` flag that also tracks this.
 *)
-type top_var_decl =
-  {tvident: string; tvtype: sizedtype; tvtrans: transformation; tvloc: string}
+type 'e top_var_decl =
+  { tvident: string
+  ; tvtype: 'e Ast.sizedtype
+  ; tvtrans: 'e Ast.transformation
+  ; tvloc: Ast.location_span sexp_opaque }
```

Comment:
Could you also make sure that all these location fields are set to compare ignore? (See Ast.ml for an example.) It's useful to be able to use comparison functions on the MIR in such a way that locations do not matter.

Reply:
I did all the rest of them, I just left this one out.